packages/backend/src/services/ProjectService.ts

```diff
@@ -61,20 +83,121 @@ export class ProjectService {
         return project;
     }
 
-    async runQuery(sql: string): Promise<Record<string, any>> {
-        return this.projectAdapter.runQuery(sql);
+    async startAdapter(projectUuid: string): Promise<ProjectAdapter> {
```

Comment:
This is the main logic to handle adapters. 

**NOTE**
* This will start an adapter the first time any function tries to obtain the adapter for a project
* The adapter will never be stopped (<- we probably want to change this soon)